draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -627,6 +627,20 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
+Moreover, if an A or AAAA response arrives before an SVCB response, a client which 
+prefers ESNI SHOULD wait up to CD milliseconds before starting secure connections 
+to either address. (Clients may begin TCP connections in this time. QUIC connections 
+should wait. This allows receipt of latent keying material in a SVCR repsonse.) 
+If an SVCB record with a "esnikeys" value and "ipv4hint" or "ipv6hint" values 
+arrives and one of those hints matches an in-progress connection, the secure 
+connection should proceed using the provided ESNI keying material. However, if 
+neither a "ipv4hint" nor "ipv6hint" is available, the client MUST resolve the 
+SvcDomainName in the SVCB response and connect to the preferred A or AAAA adddress
+according to the procedure in {{client-behavior}}.
+
+CD (Connection Delay) is a configurable parameter.  The recommended value is 50 
```

Comment:
I think this would have a very high false-positive rate due to iterative lookups required to walk the SVCB chain.

Reply:
This is the delay in waiting for the first SVCB response, not the delay in abandoning ESNI altogether. The value is meant to give the SVCB answer time to arrive since, presumably, clients would prefer using that over the A/AAAA answer(s). Maybe I’m misunderstanding your point.

Reply:
OK, got it.